Gemfile

```diff
@@ -38,3 +38,7 @@ else
   end
 end
 # END ENGINE_CART BLOCK
+
+unless File.exist?(file)
+  eval_gemfile File.expand_path('spec/test_app_templates/Gemfile.extra', File.dirname(__FILE__))
```

Comment:
I was trying to enforce consistency, but it looks like I was the one to break consistency. 🙄 https://github.com/samvera/hyrax/commit/2867afaff64180fc80df7bdd39a2a364cb356185
Let's just :shipit: